p3/data/_validation.py

```diff
@@ -33,7 +33,7 @@ def _validate_coverage_json(json_string: str) -> object:
 
     instance = json.loads(json_string)
 
-    schema_string = pkgutil.get_data(__name__, "coverage-0.1.0.schema")
+    schema_string = pkgutil.get_data(__name__, "coverage-0.2.0.schema")
```

Comment:
Are we going to keep it like this, or do we plan on changing it so that `git` tracks the schema for the next breaking release?

Realizing that doing it this way means you don't get to see the diff between schema versions.

Reply:
Good question.  I honestly hadn't thought through the ramifications of versioning the schema when we introduced it...

I think because the P3 Analysis Library is still pre-release, we have a lot of freedom here.  I'd like to keep the 0.1.0-schema around at least until Code Base Investigator transitions to the new schema, mainly so we don't have any dead links.  I think we can make a case to remove all prior versions of the schema once the P3 Analysis Library hits 1.0.0, since I hope the coverage schema will be stable by then.